_pages/dev/icon-library.md

```diff
@@ -41,17 +39,15 @@ export enum ICON_TYPE {
 }
 ```
 
-The enum is used when icons are added in the view:
+The enum is used when icons are added in the view. The following is an example:
 
 ```typescript
 <cx-icon [type]="iconType.CART"></cx-icon>
 ```
 
-
 ## Icon Configuration
-The icon configuration uses the Spartacus `ConfigModule`. The configuration allows to mix different technical icon formats in a single storefront. The configuration sample below demonstrates the usage of font based icons (using fontawesome) and SVG based icons. 
 
-This configuration uses a setup to combine (bootstrap 3) glyphicons, fontawesome (both fas and fab) and SVG icons. 
+Icons are configured in Spartacus using the `ConfigModule`. In the configuration, you can mix different technical icon formats in a single storefront. The following example demonstrates the usage of font-based icons (using Font Awesome) and SVG-based icons:
 
 ```typescript
 ConfigModule.withConfig(<IconConfig>{
```

Comment:
I discovered this week that I made a mistake in this code snippet. It should be:


```typescript
ConfigModule.withConfig(<IconConfig>{
  icon: {
    symbols: {
      CART: 'fas fa-shopping-cart',
      SEARCH: 'glyphicon glyphicon-search',
      VISA: 'fab fa-cc-visa',
      INFO: 'info',
      WARNING: 'warning'
    },
    resources: [
      {
        type: IconResourceType.LINK,
        url: 'https://use.fontawesome.com/releases/v5.8.1/css/all.css',
      },
      {
        type: IconResourceType.SVG,
        url: './assets/sprite.svg',
        types: [ICON_TYPE.INFO,ICON_TYPE.WARNING],
      },
    ]
  }
});
```

Reply:
Fixed.